### ISSUE TITLE
Details about finetuning (25 / 75 epochs)

### ISSUE THREAD

**Issue by ultmaster** [mit-han-lab/once-for-all#4]
Thanks for sharing your code for this excellent work!

Could you reveal more details about how you finetune your specialized sub-network? I didn't find the code in the repo, but hyper-parameters like batch size, optimizer, learning rate, lr decay and weight decay will be also very helpful.

Thanks again.
